src/pg_to_evalscript/javascript_datacube/DataCube.js

```diff
@@ -58,7 +58,32 @@ class DataCube {
       }
 
     removeDimension(dimension) {
```

Comment:
`removeDimension` is also called from inside `reduceByDimension` process and there it only expects to filter out the dimension and not actually drop it. It's a testament to the fact we don't have any proper tests for `reduce_dimension`...

Reply:
Should I try to fix this, or did you just mention it so I would know this?

Reply:
I changed the `reduceByDimension` function so it now only filters the dimensions instead of actually removing it